Configuration/StandardSequences/python/Eras.py

```diff
@@ -38,8 +49,10 @@ def __init__(self):
         self.Run2_2016 = cms.ModifierChain( self.run2_common, self.run2_25ns_specific, self.stage2L1Trigger )
         self.Run2_2017 = cms.ModifierChain( self.Run2_2016, self.phase1Pixel )
         # Scenarios further afield.
-        # Phase2Dev is everything for the 2023 (2026?) detector that works so far in this release.
-        self.Phase2Dev = cms.Modifier()
+        # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
+        self.Phase2 = cms.Modifier( self.phase2_common, self.phase2_tracker, self.phase2_hgc, self.phase2_muon )
+        # Phase2dev is everything for the 2023 (2026?) detector that is still in development.
+        self.Phase2dev = cms.Modifier( self.Phase2, self.phase2dev_common, self.phase2dev_tracker, self.phase2dev_hgc, self.phase2dev_muon )
```

Comment:
Just checking the phase2 eras for other reasons, but shouldn't these two be ModifierChains?